askem.dbml

```diff
@@ -128,7 +128,7 @@ Table dataset {
   id int [pk]
   name varchar [not null]
   url varchar [not null]
-  description text [not null]
+  description text 
```

Comment:
We probably want to hold off on changes to `askem.dbml` until we have some kind of migration flow set up. Would it be possible to revert this old dbml + autogen files? Is this change necessary to make GraphQL Provenance work?

Reply:
cc @mattprintz